domain/src/main/kotlin/hr/asee/android/template/domain/usecase/HomeUseCase.kt

```diff
@@ -0,0 +1,17 @@
+package hr.asee.android.template.domain.usecase
+
+import hr.asee.android.template.domain.model.common.resource.EmptyResource
+import hr.asee.android.template.domain.model.common.resource.ErrorType
+import java.time.LocalDate
+
+interface HomeUseCase {
```

Comment:
Shvaćam zašto si nazvao use case HomeUseCase, išao si prijašnjom logikom sa Loginom i Registerom. Ali use case služi za obavljanje jednog "zadatka", odnosno najmanje jedinice posla, pa bi u ovom slučaju to bilo FilterByDateUseCase ili samo FilterUseCase ako bi bilo više parametara (tipa datumi i id itd.). 
Zvali su se LoginUseCase i RegisterUseCase jer su obavljali te radnje, login i register.